tests/test.ts

```diff
@@ -165,31 +169,12 @@ describe.skip('Test clasp open function', () => {
   });
 });
 
-describe.skip('Test clasp clone function', () => {
```

Comment:
This is moved below the `clasp deploy`, `clasp deployments`, and `clasp version` tests because this removes the `.clasp.json` file, which is needed for those tests.